xml/ulp.xml

```diff
@@ -8,203 +8,270 @@
+      patches to the libraries used by a running process without interrupting
+      them. Every time a security fix is available as a live patch, customer
+      services will be secured after applying the live patch without restarting
+      the processes.
     </para>
-</note>
 
-  <para>
-   User space live patching (&ulpa;) refers to the process of applying patches
-   to the libraries used by a running process, without interrupting it. Live
-   patching operations are performed using the <systemitem>ulp</systemitem>
-   tool that is part of the <systemitem>libpulp</systemitem>.
-  </para>
+    <para>
+      Live patching operations are performed using the
+      <systemitem>ulp</systemitem> tool that is part of
+      <systemitem>libpulp</systemitem>. <systemitem>libpulp</systemitem> is a
+      framework that consists of the <systemitem>libpulp.so</systemitem>
+      library and the <command>ulp</command> binary that makes libraries live
+      patchable and applies live patches.
+    </para>
 
-  <para>
-   <systemitem>libpulp</systemitem> is a framework that enables &ulp;. It
-   consists of the <systemitem>libpulp.so</systemitem> library and tools for
-   making libraries live-patchable and applying live patches (the
-   <systemitem>ulp</systemitem> binary).
-  </para>
+    <tip>
+      <para>
+        You can run the <command>ulp</command> command either as a normal user
+        or a privileged used via the &sudo; mechanism. The difference is that
+        running <command>ulp</command> via &sudo; lets you view information of
+        processes or patch processes that are running by &rootuser;.
+      </para>
+    </tip>
 
-  <sect2 xml:id="sec-ulp-prereqs">
-   <title>Prerequisites</title>
-   <para>
-    For &ulpa; to work, two requirements must be met.
-   </para>
-   <itemizedlist>
-    <listitem os="slemicro">
-        <para>
+    <sect2 xml:id="sec-ulp-prereqs">
+      <title>Prerequisites</title>
+      <para>
+        For &ulpa; to work, two requirements must be met.
+      </para>
+      <itemizedlist>
+        <listitem os="slemicro">
+          <para>
             Install the &ulpa; on your system by running:
-        </para>
-        <screen>&prompt.root;transactional-update pkg in libpulp0 libpulp-tools</screen>
-        <para>
+          </para>
+<screen>&prompt.user;transactional-update pkg in libpulp0 libpulp-tools</screen>
+          <para>
             After successful installation, reboot your system.
+          </para>
+        </listitem>
+        <listitem os="sles;sled">
+          <para>
+            Install the &ulpa; on your system by running:
+          </para>
+<screen>&prompt.sudo;zypper in libpulp0 libpulp-tools</screen>
+        </listitem>
+        <listitem>
+          <para>
+            Applications with desired live patch support must be launched
+            preloading the <systemitem>libpulp.so.0</systemitem> library. See
+            <xref linkend="sec-ulp-libpulp"/> for more details.
```

Comment:
```suggestion
            by preloading the <systemitem>libpulp.so.0</systemitem> library.
            See <xref linkend="sec-ulp-libpulp"/> for more details.
```

Reply:
I'd be careful with the "by" here. Just the preloading itself is not launching of the app. The app is launched with the LD_PRELOAD attribute, that is what this sentence should convey :)

Reply:
yep